modules/monitoring-moving-monitoring-components-to-different-nodes.adoc

```diff
@@ -3,112 +3,102 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="moving-monitoring-components-to-different-nodes_{context}"]
-= Moving monitoring components to different nodes
+// tag::CPM[]
+[id="moving-monitoring-components-to-different-nodes-cpm_{context}"]
+= Moving monitoring components to different nodes for core platform monitoring
+// end::CPM[]
 
```

Comment:
Given the tags:
`ifndef::openshift-dedicated,openshift-rosa[]
include::modules/monitoring-moving-monitoring-components-to-different-nodes.adoc[leveloffset=+2,tags=**;CPM;!UWM]
endif::openshift-dedicated,openshift-rosa[]
include::modules/monitoring-moving-monitoring-components-to-different-nodes.adoc[leveloffset=+2,tags=**;!CPM;UWM]`

I would expect that Moving monitoring components...for core platform monitoring _not_ show up in the OCP docs...but they do https://83431--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/configuring-the-monitoring-stack.html#moving-monitoring-components-to-different-nodes-cpm_configuring-the-monitoring-stack. Is that expected on your end?

Reply:
Hello @skrthomas Thank you for the question! It _should_ show up for OCP, `ifndef` here means that it does not show for ROSA and OSD (so it's the opposite :smile: ) Since this is not the final solution (the need here is to separate the two procedures while keeping just one module for maintenance) theoretically, the ifdefs do not have to be used, because this will not show up to the users for now, but I wanted to test it here that it worked correctly when I use those. In the final product iw will be separated in a way that will not need the ifdefs :slightly_smiling_face: 

Thank you for all the questions!

Reply:
Okay, got it! Thanks for clarifications :). The combo of ifdefs and tagging made this a little bit of a brain bender 😵‍💫